libs/blocks/aside/aside.js

```diff
@@ -14,74 +14,72 @@
+import { decorateBlockBg, decorateBlockText } from '../../utils/decorate.js';
 import { createTag } from '../../utils/utils.js';
 
-const asideTypes = ['inline', 'notification'];
-const [INLINE, NOTIFICATION] = asideTypes;
-const asideSizes = ['extra-small', 'small', 'medium', 'large'];
-const [SIZE_XS, SIZE_S, SIZE_M, SIZE_L] = asideSizes;
+// standard/default aside uses same text sizes as the split
+const variants = ['split', 'inline', 'notification'];
+const sizes = ['extra-small', 'small', 'medium', 'large'];
+const [split, inline, notification] = variants;
+const [xsmall, small, medium, large] = sizes;
+const blockConfig = {
+  [split]: ['XL', 'S', 'M'],
+  [inline]: ['S', 'M'],
+  [notification]: {
+    [xsmall]: ['M', 'M'],
+    [small]: ['M', 'M'],
+    [medium]: ['S', 'S'],
+    [large]: ['L', 'M'],
+  },
+};
+
+function getBlockData(el) {
+  const variant = variants.find((variantClass) => el.classList.contains(variantClass));
+  const size = sizes.find((sizeClass) => el.classList.contains(sizeClass));
+  const blockData = variant ? blockConfig[variant] : blockConfig[Object.keys(blockConfig)[0]];
+  return variant && size && !Array.isArray(blockData) ? blockData[size] : blockData;
+}
+
+function decorateInlineVideo(el) {
+  const video = el.querySelector('a[href*=".mp4"]');
+  if (!video) return;
+  const videoHref = video.getAttribute('href');
+  const videoAttrs = videoHref.includes('autoplay=true') ? 'playsinline autoplay muted loop' : 'controls';
+  const parent = video.parentElement;
+  parent.innerHTML = `<video preload="metadata" ${videoAttrs}><source src="${video.href}" type="video/mp4" /></video>`;
+  parent.classList.add('milo-inline-video');
+  video.remove();
+}
 
 function decorateLayout(el) {
   const elems = el.querySelectorAll(':scope > div');
+  if (elems.length > 1) decorateBlockBg(el, elems[0]);
   const foreground = elems[elems.length - 1];
   foreground.classList.add('foreground', 'container');
-  if (elems.length > 1) decorateBlockBg(el, elems[0]);
-  return foreground;
-}
-
-function decorateContent(el, type, size) {
-  if (!el) return;
-  const text = el.querySelector('h1, h2, h3, h4, h5, h6, a')?.closest('div');
+  const text = foreground.querySelector('h1, h2, h3, h4, h5, h6, a')?.closest('div');
+  text?.classList.add('text');
   const picture = text?.querySelector('picture');
   const iconArea = picture ? (picture.closest('p') || createTag('p', null, picture)) : null;
-  text?.classList.add('text');
-  if (text && !text.querySelector('p')) {
-    const buttons = text.querySelectorAll('em a, strong a');
-    let btnWrap = null;
-    if (buttons[0] && !buttons[0].closest('p')) {
-      btnWrap = document.createElement('p');
-      btnWrap.append(...[...buttons].map((button) => button.parentElement));
-    }
-    const desc = createTag('p', null, text.innerHTML);
-    text.innerHTML = '';
-    text.append(
-      iconArea || '',
-      desc,
-      btnWrap || '',
-    );
-  }
```

Comment:
If I'm remembering correctly, this was added because franklin doesn't add a `p` tag if the content in a block is only one line. Did that change or are you no longer supporting a single line (for the notification aside especially)?

Reply:
@meganthecoder yes, notification asides should still supports a single line but it was easy enough to just style using the parent class name without having to wrap it in a `p` element. I was just trying to clean it up and streamline the function a bit and noticed it was easy enough to get the desired outcome without extra code, if that makes sense.